collector/eventhub/src/main/java/zipkin/collector/eventhub/EventHubCollector.java

```diff
@@ -33,15 +28,18 @@
 public class EventHubCollector implements CollectorComponent {
 
   private Builder builder;
```

Comment:
This was to allow testing the code. I can create a readonly copy of the values if it helps.